xml/System.Net.Http.Headers/AuthenticationHeaderValue.xml

```diff
@@ -75,7 +75,21 @@
       <Docs>
         <param name="scheme">The scheme to use for authorization.</param>
         <summary>Initializes a new instance of the <see cref="T:System.Net.Http.Headers.AuthenticationHeaderValue" /> class.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+<format type="text/markdown"><![CDATA[ 
+Example:
+```cs
```

Comment:
We should have at a minimum, C# and VB.NET examples in here

Reply:
nit: Additionally, I think we use **csharp** as the language identifier, not **cs**. But I think **cs** works.

Reply:
@Thraka done